addons/website/static/src/scss/website.scss

```diff
@@ -200,9 +197,22 @@ $-seen-urls: ();
     .o_menu_loading {
         flex-wrap: nowrap !important;
         overflow: hidden !important;
+        // The "flex-wrap + overflow" style on will only hide the overflowed portion
+        // of the last visible element (not the entire item).
+        // That's why we need to initially hide menu items to reduce flickering.
+        opacity: 0;
```

Comment:
This is exactly the opposite of what we want (I hope you understood that in the spec ^^) -> we want the menu to be there as soon as the page is visible (I tested on runbot and still see the menu flashing, that's the first thing to remove in the code and build your work over there).

Reply:
Yes I understand, but (at least with the current code) we can't get the same thing (only using XML & CSS) as after the `autohideMenu` option, so I thaugh about limiting the delay using CSS animation.